example/example2.rb

```diff
@@ -20,15 +20,15 @@
   puts ""
   puts "call #{i} ..."
   out = Vips::Operation.call "black", [200, 300]
-  if out.width != 200 or out.height != 300
+  if (out.width != 200) || (out.height != 300)
```

Comment:
I'd remove the brackets, I've not used them anywhere else.

Reply:
To explain: the whole of ruby-vips does `xxx || yyy`, so if we add brackets here, we should add them everywhere.

Personally, I don't like brackets in chained `||` and `&&` expressions, they add noise and reduce clarity (imo). If you have an expression that mixes `&& and `||` then you need them, but that types of conditional is rare (and probably too complicated anyway).

I think I would change this to:

```ruby
  if out.width != 200 || out.height != 300
```

Reply:
@jcupitt updated